resources/helm/v2.2/istio-telemetry/kiali/templates/kiali-cr.yaml

```diff
@@ -28,7 +28,8 @@ spec:
 {{- end }}
 {{- if .Values.kiali.ingress }}
 {{- if .Values.kiali.ingress.enabled }}
-    ingress_enabled: {{ .Values.kiali.ingress.enabled }}
+    ingress:
```

Comment:
Did you update this file manually?

Reply:
Yes. Is this `resources/helm/v2.2/istio-telemetry/kiali/templates/kiali-cr.yaml` auto-generated somehow?

Reply:
Yes, they are auto-generated. You just need to touch the file in the `overlay` directory and run `make gen`.

If you need to change the 2.2 resources as well, for the ingress stuff, you need to do that in the `maistra-2.2` branch. Once it's merged there, it will be copied to the 2.3 branch.

Reply:
OK.  I won't mess with the 2.2 one then. It isn't important. This will still work. Going forward, the new overlay will use the new ingress setting.